### PR TITLE
Navbar restructure files

### PR DESCRIPTION
**Pull request by ainneo**
BootStrap5 and Sass are fully functionally and working! Yay!
Nav component has been implemented and the landing page has been updated.
I double checked all browsers, so everything should be responsive,  but if you get a moment it would be nice if you could also check on your computer.

We need to work on meta tags for SEO, so I will created another issue ticket to fix that.
We also need a footer component... that could be a good first issue for someone, now that everything is setup.

### CODE REVIEW

File: components/Layout.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import Nav from './Nav'
+import React, { ReactNode } from 'react'
+// import Meta from "./Meta";
```
Comment (marianebk) [inline components/Layout.tsx:3]: Commented code. Should we keep it?

File: pages/_app.tsx [hunk 2]
```diff
@@ -1,10 +1,32 @@
-import { AppProps } from 'next/app'
 import '../styles/globals.scss'
-
+// import 'bootstrap/dist/css/bootstrap.min.css'
```
Comment (marianebk) [inline pages/_app.tsx:2]: commented out code. Should we keep it?

File: pages/_app.tsx [hunk 3]
```diff
@@ -1,10 +1,32 @@
...
+          src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.2/dist/umd/popper.min.js"
+          integrity="sha384-IQsoLXl5PILFhosVNubq5LC7Qb9DXgDA9i+tQ8Zj3iwWAwPtgFTxbJ8NT4GN1R8p"
+          crossOrigin="anonymous"></script>
+        <script
+          src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/js/bootstrap.min.js"
+          integrity="sha384-Atwg2Pkwv9vp0ygtn1JAojH0nYbwNJLPhwyoVbhoPwBhjQPR5VtM2+xf0Uwh9KtT"
+          crossOrigin="anonymous"></script>
+        {/* <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+        <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js" integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"></script>
+        <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/js/bootstrap.min.js" integrity="sha384-w1Q4orYjBQndcko6MimVbzY0tgp4pWB4lZ7lr30WKz0vr/aWKhXdBNmNb5D92v7s" crossorigin="anonymous"></script>
```
Comment (marianebk) [inline pages/_app.tsx:24]: commented out code. Should we keep it?

**Review by marianebk — APPROVED** [review]

**Review by marianebk — DISMISSED** [review]

File: pages/_app.tsx (hunk 2 above)
Comment (ainneo) [reply pages/_app.tsx:2]: We can delete this...

File: pages/_app.tsx (hunk 3 above)
Comment (ainneo) [reply pages/_app.tsx:24]: I want to keep this. This is for bootstrap4... incase someone wants to use things from 4 instead of 5.

**Review by marianebk — APPROVED** [review]

**Review by marianebk — APPROVED** [review]

**Pull request merged**